2022-RapidReact/src/main/java/frc/robot/commands/ActivateConveyor.java

```diff
@@ -4,23 +4,18 @@
 
 package frc.robot.commands;
 
-import frc.robot.subsystems.ExampleSubsystem;
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.StorageSystem;
 
-/** An example command that uses an example subsystem. */
-public class ExampleCommand extends CommandBase {
-  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
-  private final ExampleSubsystem m_subsystem;
-
-  /**
-   * Creates a new ExampleCommand.
-   *
-   * @param subsystem The subsystem used by this command.
-   */
-  public ExampleCommand(ExampleSubsystem subsystem) {
-    m_subsystem = subsystem;
+public class ActivateConveyor extends CommandBase {
+  private final StorageSystem storageSystem;
+  private double convSpeed;
+  /** Creates a new activateConveyor. */
+  public ActivateConveyor(StorageSystem st, double speed) {
     // Use addRequirements() here to declare subsystem dependencies.
-    addRequirements(subsystem);
+    convSpeed = speed;
+    storageSystem = st;
```

Comment:
Talvez, fique mais legível se o nome para o "st" for mais completo, acredito que seja difícil identificar o que ele faz se ele for visto no meio do código.